templates/etc/systemd/system/docker-run.erb

```diff
@@ -39,6 +39,9 @@ ExecStart=/usr/bin/<%= @docker_command %> run \
         <% if @command %> <%= @command %><% end %>
 <%- if @before_stop %>ExecStop=-<%= @before_stop %>
 <% end -%>
+<% if @stop_before_kill -%>
+ExecStop=-/usr/bin/<%= @docker_command %> stop <% if @stop_timeout != false %>-t <%= @stop_timeout %> <% end %><%= @sanitised_title %>
+<%- end -%>
 ExecStop=-/usr/bin/<%= @docker_command %> kill <%= @sanitised_title %>
```

Comment:
Hi @lkosz, killing after stop is unnecessary, because `docker stop` will [kill container after timeout](https://docs.docker.com/engine/reference/commandline/stop/).

Configuring stop timeout seems reasonable for me, but I think it will be better to remove this line with `docker kill`.